### ISSUE TITLE
[1.3] Importing file from another imported file fails in a package

### ISSUE THREAD

**Issue by kookoskar** [meteor/meteor#5957]
[Example repo.](https://github.com/kookoskar/meteor-bug-example) Throws the following:
```
W20160108-16:12:59.144(2)? (STDERR) /home/oskar/development/meteor-example/.meteor/local/build/programs/server/packages/bug-example.js:33
W20160108-16:12:59.144(2)? (STDERR) import imported2 from './imported2.js'                                        
W20160108-16:12:59.145(2)? (STDERR) ^^^^^^
W20160108-16:12:59.145(2)? (STDERR) SyntaxError: Unexpected reserved word
W20160108-16:12:59.145(2)? (STDERR)     at /home/oskar/development/meteor-example/.meteor/local/build/programs/server/boot.js:241:30
W20160108-16:12:59.145(2)? (STDERR)     at Array.forEach (native)
W20160108-16:12:59.145(2)? (STDERR)     at Function._.each._.forEach (/home/oskar/.meteor/packages/meteor-tool/.1.1.11-modules.3.14elear++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160108-16:12:59.145(2)? (STDERR)     at /home/oskar/development/meteor-example/.meteor/local/build/programs/server/boot.js:137:5
```

I'm not exactly sure what is causing this. Maybe it's the file structure. It works when `imported1.js` and `imported2.js` are not in the `client/` directory.

**Comment by kookoskar** [comment]
It also works when:
* `client.js` is in the `client/` directory.
* `client.js` imports from a directory **not** named `client/`

**Comment by kookoskar** [comment]
This is fixed in `METEOR@1.3-modules-beta.3`.

Closing the issue.
